happi/client.py

```diff
@@ -535,3 +535,34 @@ def find_config():
         # If found nothing
         raise EnvironmentError("No happi configuration file found. "
                                "Check HAPPI_CFG.")
+
+    def list_choices(self, field):
```

Comment:
Trying to think of a name for this ... `list_choices` is kind of vague but I can't think of anything better. @ZLLentz @klauer?

Reply:
Maybe:
```suggestion
    def choices_for_field(self, field):
```

Reply:
Think I like it better

Reply:
`choices_for_field`
`choices`
`field_choices`
`options_for_field`
`options`
`field_options`

Reply:
Changed in 2191d86